README.md

```diff
@@ -39,10 +39,29 @@ $ ./gradlew docker
 
 You can run the fat jar as follows.
 
+
+```console
+
+$ java -jar scalar-admin-<version>-all.jar -c <COMMAND> -a <IP1:Port1,IP2:Port2,...>
+```
+
+The command above can take effect on multiple ScalarDB or ScalarDL instances in one shot.
+
+**NOTE** If you previously used the following command to pause and unpause Scalar pods in Kubernetes clusters,
+we recommend switching to [Scalar Admin for Kubernetes](https://github.com/scalar-labs/scalar-admin-for-kubernetes).
```

Comment:
At this time, I think we should recommend the `Scalar Admin for Kubernetes` in spite of whether using the `-s` option or not. This is because we recommend using Kubernetes as a platform for Scalar products and the `Scalar Admin` is error-prone in the Kubernetes environment.